src/localization/buttonStrings.js

```diff
@@ -27,9 +27,9 @@ export const buttonStrings = new LocalizedStrings({
     view_regimen_data: 'View Regimen Data',
   },
   fr: {
-    // TODO: view_regimen_data
+    view_regimen_data: 'Données de régimes',
     add_batch: 'Ajouter un lot',
-    add_master_list_items: "Ajouter une liste-Maître d'articles ?",
+    add_master_list_items: 'Ajouter à\npartir des\nlistes maîtres',
```

Comment:
Noticed this translation has changed. Just checking this is the correct translation?

Reply:
oops sorry I meant to put that in the top comment. I took this translation for desktop explicitly. The `?` tipped me off it may not have been translated correctly:

Original: Ajouter une liste-Maître d'articles -- Add a master list of articles
New: Ajouter à partir des listes maîtres -- Add from master lists

@alainsussol this is correct? :)

Reply:
Google translate says you're right 😄 . For sake of getting `v3.0.0` out, will merge (can always fix later, if needed 👍 ).

Reply:
Yes this is correct...